src/js/edu-benefits/utils/validations.js

```diff
@@ -44,6 +44,10 @@ function isValidYear(value) {
   return Number(value) >= 1900;
 }
 
+function isValidMonths(value) {
+  return Number(value) >= 0;
```

Comment:
This might make more sense as `isValidPositiveNumber`. It's for a standalone months field.